package.json

```diff
@@ -43,22 +43,22 @@
     "cordova-common": "^3.1.0",
     "execa": "^2.0.1",
     "expect-telnet": "^1.0.0",
-    "jasmine": "^3.4.0",
+    "jasmine": "^3.6.1",
```

Comment:
Hopefully this doesn't have problems like it did in iOS repo which got reverted.

Reply:
Hopefully not, but the iOS test (expect the flaky 12.0) pass - so we _should(?)_ be good.